release_notes_generator/generator.py

```diff
@@ -79,18 +79,26 @@ def generate(self) -> Optional[str]:
         rls = self._safe_call(repo.get_latest_release)()
         if rls is None:
             logger.info("Latest release not found for %s. 1st release for repository!", repo.full_name)
+        else:
+            logger.debug("RLS created_at: %s, published_at: %s", rls.created_at, rls.published_at)
 
         # default is repository creation date if no releases OR created_at of latest release
         since = rls.created_at if rls else repo.created_at
         if rls and ActionInputs.get_published_at():
             since = rls.published_at
 
-        issues = self._safe_call(repo.get_issues)(state=ISSUE_STATE_ALL, since=since)
+        issues = issues_all = self._safe_call(repo.get_issues)(state=ISSUE_STATE_ALL, since=since)
```

Comment:
Please add a simple comment for this block of code, to simply describe, why you keep the original list and one for modifying.

Reply:
Fixed in commit 317e9eca1b8d02d7fe8570b1620f676fb94163cb.